tiledb/api/src/array/attribute.rs

```diff
@@ -201,12 +206,23 @@ impl<'ctx> Attribute<'ctx> {
     }
 }
 
-impl<'ctx> From<(&'ctx Context, RawAttribute)> for Attribute<'ctx> {
-    fn from(value: (&'ctx Context, RawAttribute)) -> Self {
-        Attribute {
-            context: value.0,
-            raw: value.1,
-        }
+impl<'ctx> Debug for Attribute<'ctx> {
```

Comment:
PBT strategy requires Debug.

I chose to emit some fields as JSON since I firmly believe that this sort of thing should be easily processed by a script.

Reply:
Definitely agree on JSON. Hand generating for the moment seems fine. I'm not sure how heavy of a dependency it is, but I'm guessing we'll probably get some side eye from not using serde_json eventually.